test/uk/gov/hmrc/tai/viewModels/TaxCodeViewModelSpec.scala

```diff
@@ -214,5 +334,6 @@ class TaxCodeViewModelSpec extends PlaySpec with FakeTaiPlayApplication with I18
   private val scottishTaxRateBands = Map.empty[String, BigDecimal]
 
   def sut(taxCodeIncomes: Seq[TaxCodeIncome]) = TaxCodeViewModel(taxCodeIncomes, scottishTaxRateBands)
```

Comment:
While we're in this file can we rename this to something more meaningful?

Reply:
Agreed, it's done now